library/kotlin/src/io/envoyproxy/envoymobile/EnvoyStreamEmitter.kt

```diff
@@ -29,22 +29,14 @@ class EnvoyStreamEmitter(
     return this
   }
 
-  /**
-   * For ending an associated stream and sending trailers.
-   *
-   * @param trailers to send with ending a stream. If null, stream will be closed with an empty data frame.
-   */
-  override fun close(trailers: Map<String, List<String>>?) {
-    trailers?.let {
-      stream.sendTrailers(it)
-      return
-    }
-    stream.sendData(ByteBuffer.allocate(0), true)
+  override fun close(trailers: Map<String, List<String>>) {
```

Comment:
Not going to keep the comments?

Reply:
They're specified in the superclass, so I figured it was better to have them in one place rather than copy-pasted to every implementation. WDYT?

Reply:
Could we also delete the other comments? If it's a lot to change, we can both do it in another PR

Reply:
I'll do it in a follow up PR yep, waiting on this to merge to avoid conflicts

Reply:
https://github.com/lyft/envoy-mobile/pull/799/files#diff-9652aa5519ef4ba01992a74aaa5cd9b1L10